linkerd/app/outbound/src/endpoint.rs

```diff
@@ -319,30 +303,13 @@ impl<S> Outbound<S> {
 
 // === impl EndpointError ===
 
-impl fmt::Display for EndpointError {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let Self {
-            addr,
-            protocol,
-            source,
-        } = self;
-        let protocol = match protocol {
-            Some(http::Version::Http1) => "HTTP/1",
-            Some(http::Version::H2) => "HTTP/2",
-            None => "opaque",
-        };
```

Comment:
the reason that I included protocols here is because the `EndpointError` may be handled by a `ClientRescue` layer without being wrapped in other stacks' higher-level error contexts, so i thought it was desirable to include that information here. i'm fine with removing it for simplicity's sake, though.

Reply:
ooooh. hmm. okay. I need to refresh on that. In any case, the target types are going to change, so we can keep it simple for now and then polish as it settles.